docs/administration/managing-infrastructure/script-console.md

```diff
@@ -42,6 +44,10 @@ Besides making it easy to run a script on many servers, the other advantage of u
 
 ![](images/3277919.png)
 
+## Targeting the Octopus Server
+
+You cannot target the Octopus Server with the Script Console. If you want to run ad-hoc tasks on your Octopus Server, you should install a Tentacle agent on your Octopus Server and register that Tentacle as a worker or deployment target.
```

Comment:
Are there any caveats we should add to this? i.e., is it a good idea to install Tentacle on the Octopus Server?

Reply:
@wordlee Yep, that is our recommended approach as it is more secure than running processes on the built in worker.

Ideally you want it off server entirely, however this is a good baseline/starting point without getting too complex